mmf/utils/wandb_logger.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+from typing import Dict, Optional
+
+from mmf.utils.distributed import is_master
+
+
+try:
+    import wandb
```

Comment:
Move this check inside the init of `WandbLogger`. In exception, you can then throw a warning to install wandb and die.

Reply:
Fixed.